pipeline/enigma/enigma-merge_hg38.py

```diff
@@ -0,0 +1,70 @@
+'''
+This script takes processed ENIGMA files and concatenates them.
+A list of files can be taken as a command line argument or the
+default using the output file naming convention from the processing scripts
+'''
+
+import glob
+import numpy as np
+import sys
+import re
+import argparse
+import os
+import datetime
+
+COLUMNS = np.array(["Gene_symbol",
+                    "Genomic_Coordinate",
+                    "Reference_sequence",
+                    "HGVS_cDNA",
+                    "BIC_Nomenclature",
+                    "Abbrev_AA_change",
+                    "URL",
+                    "Condition_ID_type",
+                    "Condition_ID_value",
+                    "Condition_category",
+                    "Clinical_significance",
+                    "Date_last_evaluated",
+                    "Assertion_method",
+                    "Assertion_method_citation",
+                    "Clinical_significance_citations",
+                    "Comment_on_clinical_significance",
+                    "Collection_method",
+                    "Allele_origin",
+                    "ClinVarAccession",
+                    "HGVS_protein"])
+
+
+def main():
+    default_input_files = sorted(glob.glob("output/*last_updated*hg38*tsv"))
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--input_folder', help='Link to folder with all processed Enigma files to merge')
+    parser.add_argument('-o', '--output_folder', help='Folder to contain output file')
+    parser.add_argument('-f', '--input_files', nargs="*",
+                        help='Links to processed ENIGMA files to be merged if input folder is not provided.',
+                        default=default_input_files)
+    args = parser.parse_args()
```

Comment:
Moving forward, the next time we get new ENIGMA data, we could change this script to get the input files directly from Synapse.  That would save one manual step.